packages/bpk-component-theme-toggle/src/theming.js

```diff
@@ -50,6 +50,12 @@ const bpkCustomTheme = {
   buttonSecondaryHoverBackgroundColor: '#ffffff',
   buttonSecondaryActiveBackgroundColor: '#ffffff',
 
+  linkColor: '#865f9e',
+  linkTextColor: '#865f9e',
+  linkHoverColor: '#461962',
+  linkActiveColor: '#2d244c',
+  linkVisitedColor: '#1a1331',
+
```

Comment:
As the PR for refactoring this file to use variables has been merged, can you update this to use them too?